core/src/main/java/feast/core/config/FeastProperties.java

```diff
@@ -278,5 +280,19 @@ public void validate() {
                 + e.getMessage());
       }
     }
+
+    // Validate AuthenticationProperties
+    Set<ConstraintViolation<AuthenticationProperties>> authenticationPropsViolations =
```

Comment:
How do we know whether we are actually validating anything here and whether failures are being detected?

Reply:
This is validating `OneOfStrings` validation annotations we have on providers for authentication and authorization.